components/notability/commons/notability_checker.lua

```diff
@@ -184,12 +184,21 @@ function NotabilityChecker._calculateWeightForTournament(tier, tierType, placeme
 		Config.weights, function(tierWeights) return tierWeights['tier'] == tier end
 		)
 
+	if not weightForTier then
+		return 0
+	end
+
 	local tierPoints = Array.find(
 		weightForTier['tiertype'],
 		function(pointsForType)
 			return pointsForType['name'] == (tierType or Config.TIER_TYPE_GENERAL)
 		end
 	)['points']
```

Comment:
Is a check needed here as well?

Reply:
i already added one

Reply:
I mean before `.points` (ie if Array.find return nil rather than table, if it can)

Reply:
it can, yeah good catch